docs/articles/vs-test-adapter/Adapter-Installation.md

```diff
@@ -4,43 +4,63 @@ uid: vstestadapterinstallation
 
 # Adapter Installation
 
-There are two ways of installing the adapter within Visual Studio. See below for info on how to choose.
-
-* Add it as a NuGet package to your solution. (Requires VS 2012 Update 1 or later)
-* Use the Extension Manager  (Deprecated and pending future removal; see notes below.)
+The adapter is to be installed as a nuget package to each of your test projects.
 
 ## Installing the NuGet Package
 
-To add it is a NuGet package, you must have an active solution, then follow these steps:
+You will most likely want to add a new NUnit Test Project to your solution.  The easiest way is to use the ```dotnet``` command on the command line.
 
-1. From Tools menu, use Library Package Manager and select Manage NuGet packages for solution
-2. In the left panel, select Online
-3. Locate (search for) NUnit 3.0 Test Adapter in the center panel and highlight it
-4. Click 'Install'
-5. In the "Select Projects" dialog, you need to select at least one project to add the adapter to, see notes below.
+Go to your project root and where you want to add your project.
+
+Create a folder for the project with a name matching what your new test project should be named.
+
+Then run the command ````dotnet new nunit```
+
+You will now get a complete nunit test project with the same name as the folder.  You will also have a template unit test class there as a starter.
 
-## Installing With the Extension Manager
+If you have a Visual Studio solution file in the root folder, you can go there and add the new nunit csproj easily:
 
-> [!WARNING]
-> The use of the VSIX extension manager is deprecated.  It still works, but at some time support for it will be removed from Visual Studio, and also newer versions of the adapter. We recommend you try to move to nuget based adapters.
+Assume you are at a solution root, and you either have a solution file, or have just been adding it with ```dotnet new sln```
 
-To install the NUnit Test Adapter using the Extension Manager, follow these steps:
+```cmd
+md Whatever.Test
+cd Whatever.Test
+dotnet new nunit
```

Comment:
💡 Should we add a note here about the fact that there are arguments and options to consider and running this command as-is may not produce the output folks are looking for, e.g. project name? Or should we maybe be more explicit with a comment that the project will be named after the directory?

Reply:
To me it can be either way, we're not going to do a complete course on dotnet, solutions and so, - on the other hand, it doesn't cost much to add what you suggest.  So, feel free :-)